submissions/serializers.py

```diff
@@ -90,6 +90,14 @@ class ScoreSerializer(serializers.ModelSerializer):
     # Ensure that the created_at datetime is not converted to a string.
     created_at = DateTimeField(format=None, required=False)
 
+    annotation_types = serializers.SerializerMethodField()
+    def get_annotation_types(self, obj):
+        """
+        Inspect ScoreAnnotations to attach the annotation_type of any found annotations.
+        """
+        annotations = ScoreAnnotation.objects.filter(score_id=obj.id)
+        return [annotation.annotation_type for annotation in annotations]
```

Comment:
I question making this a list. If there are actually multiple annotations, knowing which one is related to a specific "annotation_type" seems like it would be very useful. I would think that a dict would make more sense, with the key being specific to the annotation (I don't know what annotation looks like off-hand).